static/app/stores/guideStore.tsx

```diff
@@ -225,7 +226,7 @@ const storeConfig: GuideStoreDefinition = {
    *  - If the user has already seen the guide, don't show the guide
    *  - Otherwise show the guide
    */
-  updateCurrentGuide() {
+  updateCurrentGuide(dismissed?: boolean) {
```

Comment:
When is this called with dismissed = true?

Reply:
@Neo-Zhixing when the user dismisses a guide